setup.py

```diff
@@ -43,6 +43,7 @@
         "MutatorMath>=2.0.2",
         "defcon>=0.2.1",
         "booleanOperations>=0.6.4",
+        "mock>=2.0.0",
```

Comment:
I created a test-requriements.txt file and probably I'm changing our test engine config.